tests/lib/rules/prefer-user-event.test.ts

```diff
@@ -130,5 +191,53 @@ ruleTester.run(RULE_NAME, rule, {
         errors: [{ messageId: 'preferUserEvent' }],
       })
     ),
+    ...createScenarioWithImport<InvalidTestCase<MessageIds, Options>>(
+      (libraryModule: string, fireEventMethod: string) => ({
+        code: `
+        const { fireEvent } = require('${libraryModule}')
+        fireEvent.${fireEventMethod}(foo)
+      `,
+        errors: [{ messageId: 'preferUserEvent' }],
```

Comment:
It would be nice to check line and column of the error.

Reply:
added in https://github.com/testing-library/eslint-plugin-testing-library/pull/251/commits/06cb9cf45c0a78f963461dcaeb3230840ed657e0